hazelcast/src/main/java/com/hazelcast/client/HazelcastClient.java

```diff
@@ -68,7 +67,7 @@
 public final class HazelcastClient {
 
     private static final AtomicInteger CLIENT_ID_GEN = new AtomicInteger();
-    private static final ConcurrentMap<String, InstanceFuture<HazelcastClientProxy>> CLIENTS = new ConcurrentHashMap<>(5);
+    private static final ConcurrentHashMap<String, InstanceFuture<HazelcastClientProxy>> CLIENTS = new ConcurrentHashMap<>(5);
```

Comment:
What is the rationale for this?

Reply:
Nothing in particular. I just like this version better. I can take this into the previous version if you want